packaging/material/exercise_packaging_part1.md

```diff
@@ -0,0 +1,14 @@
+# Exercise: Packaging Python code Part 1
```

Comment:
The file name should end with `_text.md` such that the GitHub Action can identify it as text file when converting it to PDF.